polylogue/cli/examples.py

```
@@ -0,0 +1,88 @@
+"""Shared click/argparse command examples."""
+
+from __future__ import annotations
+
+from typing import Dict, List, Tuple
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)